docs/verify/simulate-with-qiskit-aer.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<p>for documentation we use the second person voice ("you can do this.." not "one can do this...")</p><p> </p><p>Also we shouldn't call this page a "guide" as i think ti will conflict with what they're calling things in the learning section 🙄 I'd recommend saying "example" instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I don't think "example" is good since it suggests something smaller than the entire page. What about "page"?

Reply:
I wrote "page" for now.

Reply:
<p>can you also change "In practice, you can employ a variety of error mitigation techniques to counter the effects of the noise, but a discussion of these techniques is outside the scope of this guide" to page pls?</p>

Reply:
That sounded really weird so I changed both to "article." Hope that's okay.